File: DataSummitSolution/DataSummitUI/src/app/drawings/models/sentence.ts
import { Rectangle } from "../../profileVersion/create/models/rectangle";
import { Drawing } from "./drawing";
import { Property } from "../../properties/models/property";

export class Sentence {

	SentenceId: number;
	Words: string;
	RectangleId: number;
	Vendor: string;
	IsUsed: boolean;
	Confidence?: number;
	DrawingId: number;
	Drawing: Drawing;
	Properties: Property[];
	Rectangles: Rectangle[];

	constructor(sentenceId?: number, words?: string, rectangleId?: number,
		vendor?: string, isUsed?: boolean, confidence?: number,
		drawingId?: number, drawing?: Drawing, properties?: Property[],
		rectangles?: Rectangle[])
	{
		this.SentenceId = sentenceId || 0;
		this.Words = words || "";
		this.RectangleId = rectangleId || 0;
		this.Vendor = vendor || "";
		this.IsUsed = isUsed || false;
		this.Confidence = confidence || null;
		this.DrawingId = drawingId || 0;
		this.Drawing = drawing;
		this.Properties = properties;
		this.Rectangles = rectangles;
	}
}

File: DataSummitSolution/DataSummitModels/DB/Sentence.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace DataSummitModels.DB
{
    public partial class Sentence
    {
        public Sentence()
        {
            Properties = new HashSet<Property>();
        }

        public Guid SentenceId { get; set; }
        public string Words { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public int Left { get; set; }
        public int Top { get; set; }
        public string Vendor { get; set; }
        public bool IsUsed { get; set; }
        public decimal? Confidence { get; set; }
        public decimal? SlendernessRatio { get; set; }
        public long DocumentId { get; set; }
        public string ModifiedWords { get; set; }

        public virtual Document Document { get; set; }
        public virtual ICollection<Property> Properties { get; set; }
    }
}
